packages/api-gateway-service/package-lock.json

```diff
@@ -15030,8 +15030,7 @@
               "version": "2.1.1",
```

Comment:
@anjneeksharma - Can you please check out this file becuse this is not related to feedback schema changes

Reply:
yes while rebasing it got some update on that